pkg/jx/cmd/get_apps.go

```diff
@@ -93,21 +129,58 @@ func (o *GetAppsOptions) Run() error {
 		return nil
 	}
 
-	table := o.generateTable(apps, kubeClient)
+	if o.ShowStatus {
+		if checkDifferentAppsInAppsSlice(apps) {
+			fmt.Fprint(o.Out, "Different apps provided. Provide only one app to check the status of\n")
+			return nil
+		}
+		return o.generateAppStatusOutput(&apps.Items[0])
+	}
 
+	if o.Output != "" {
+		appsResult := o.generateTableFormatted(apps)
+		return o.renderResult(appsResult, o.Output)
+	}
+	table := o.generateTable(apps, kubeClient)
 	table.Render()
 	return nil
 }
 
+func (o GetAppsOptions) generateAppStatusOutput(app *v1.App) error {
```

Comment:
I guess, you missed a pointer on the receiver.

Reply:
works as well, the conversion is implicit. A pointer receiver might be better of course and consistent with the rest. Good catch.

Reply:
That's right but is good to have a consistent style. e.g. the linter will flag this inconsistency if the pointer are mixed with non-pointer receivers.